coderd/database/migrations/000304_template_version_terraform_values.up.sql

```diff
@@ -0,0 +1,5 @@
+create table template_version_terraform_values (
+	template_version_id uuid not null unique references template_versions(id),
+	updated_at timestamptz not null default now(),
+	cached_plan jsonb not null
+);
```

Comment:
Attaching this to the version rather than the job makes this more intuitive and correct if we add refreshing :+1:.